openapi-generator/src/main/resources/fullstory-typescript/api-single.mustache

```diff
@@ -40,7 +40,7 @@ export class {{classname}} {
      * @summary {{&summary}}
      {{/summary}}
      {{#allParams}}
-     * @param {{paramName}}{{#description}} {{.}}{{/description}}
+     * @param {{paramName}}{{#unescapedDescription}} {{&.}}{{/unescapedDescription}}{{^unescapedDescription}} {{&description}}{{/unescapedDescription}}
```

Comment:
We don't really want escaped texts here. But for some reason some of the models didn't have any `unescapedDescription`s, so just fall back to the `description` if any. (should probably figure out why at some point..)